client/src/pages/Login.js

```diff
@@ -0,0 +1,61 @@
+import { defaults } from "pg";
```

Comment:
Note that the overwhelming majority of the changes in this PR are because you included this line, trying to import `pg` into the client-side code. This dependency is for database access, it doesn't make sense to be doing that directly from React (that's what the Express API is for). This was then (correctly!) removed again in the second commit but all of the Webpack configuration and additional dependencies remained.

Trying this out locally it seems you'd get e.g.:

```
ERROR in ./node_modules/pg-protocol/dist/parser.js 9:33-50
Module not found: Error: Can't resolve 'assert' in 'path/to/starter-kit/node_modules/pg-protocol/dist'

BREAKING CHANGE: webpack < 5 used to include polyfills for node.js core modules by default.
This is no longer the case. Verify if you need this module and configure a polyfill for it.
```
"Verify if you need this module" is the key here - you don't! I'll add this to the wiki, but please do bring issues up in the support channel if you're getting outputs you don't understand.

Reply:
Hi @textbook , yes I had  lots of confusion while running this template so I tried too many changes on my file, that's what it happened . Thank you so much for pointing these mistakes.